samba/data/public/index.html

```diff
@@ -0,0 +1,352 @@
+        }
+
+        [type=search]::-webkit-search-decoration {
+            -webkit-appearance: none
+        }
+
+        ::-webkit-file-upload-button {
+            -webkit-appearance: button;
+            font: inherit
+        }
+
+        details {
+            display: block
+        }
+
+        summary {
+            display: list-item
+        }
+
+        template {
+            display: none
+        }
+
+        [hidden] {
+            display: none
+        }
+    </style>
+    <style>
+        body {
+            background-color: #1D1B1B;
+            font-family: system-ui, -apple-system, BlinkMacSystemFont, Roboto, Helvetica Neue, Segoe UI, Arial, Noto Sans, sans-serif, Apple Color Emoji, Segoe UI Emoji, Segoe UI Symbol, Noto Color Emoji;
+            color: #fff;
+        }
+
+        *,
+        h1,
+        h2,
+        h3,
+        h4,
+        h5,
+        h6,
+        p,
+        span {
+            color: #fff;
+            font-size: 20px;
+            font-weight: normal;
+        }
+
+        .success {
+            color: #63FB72;
+        }
+
+        .text-muted {
+            opacity: 0.8;
+        }
+
+        .text-small {
+            font-size: 14px;
+        }
+
+        hr {
+            width: 100%;
+            height: 2px;
+            background: #2F2C2C;
+            border: none;
+            margin: 40px 0;
+        }
+
+        .container {
+            padding: 40px;
+            max-width: 1440px;
+            margin: auto;
+        }
+
+        .container.terminal-container {
+            background-color: #101010;
+            border-radius: 30px;
+            margin-bottom: 40px;
+        }
+
+        .app-icon {
+            border-radius: 20px;
+        }
+
+        .app {
+            display: flex;
+            margin: 20px 0 40px 0;
+        }
+
+        .app>.app-icon {
+            flex-shrink: 0;
+            height: 140px;
+            width: 140px;
+            box-shadow: 0 0 40px 0 rgba(0, 0, 0, 0.95);
+            margin-right: 24px;
+        }
+
+        .app>.app-details>.app-status {
+            display: block;
+            font-size: 20px;
+            margin: 10px 0 0 0;
+        }
+
+        .app>.app-details>.app-name {
+            font-size: 52px;
+            line-height: 52px;
+            font-weight: bold;
+            margin: 10px 0 0 0;
+        }
+
+        #terminal>iframe {
+            border: 0px !important;
+            position: relative !important;
+            height: 70vh !important;
+            width: 100% !important;
+            max-width: 1360px !important;
+        }
+
+        /* Firefox iframe-ception fix */
+
+        body.iframe {
+            background-color: #101010;
+        }
+
+        body.iframe .app-container {
+            display: none;
+        }
+
+        body.iframe .container.terminal-container {
+            border-radius: 0;
+            padding: 0;
+            margin: 0;
+        }
+    </style>
```

Comment:
Some of the classes are not needed anymore, because we don't use them for example for the terminal. Please remove all non-needed classes